### DISCUSSION TITLE
[@types/eventsource] is missing status attribute in MessageEvent

### DISCUSSION THREAD

**Discussion by orange-buffalo** [DefinitelyTyped/DefinitelyTyped#62401] [Issues with a @types package]
[EventSource/eventsource ](https://github.com/EventSource/eventsource) library extends the standard message event with additional `status` attribute:
> Unauthorized and redirect error status codes (for example 401, 403, 301, 307) are available in the status property in the error event.
```
es.onerror = function (err) {
  if (err) {
    if (err.status === 401 || err.status === 403) {
      console.log('not authorized');
    }
  }
};
```

However, its types module `@types/eventsource` does not define `status` on `MessageEvent` and so it is not possible to check for the error response status.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #62529

**Reply by orange-buffalo** [reply]
Thanks!
